src/redux/actions/pipeline/runGem2s.js

```diff
@@ -13,8 +13,9 @@ import {
 
 import loadBackendStatus from '../backendStatus/loadBackendStatus';
 
-const runGem2s = (experimentId) => async (dispatch, getState) => {
-  const { experiments } = getState();
+const runGem2s = (experimentId, gem2sHash = 'hash-not-defined') => async (dispatch, getState) => {
```

Comment:
In what cases will we have not defined hash value?

Reply:
When we `reRunGems()` from the GEM2S loading screen. The user can not edit the project again, but we need GEM2S to re-run with the same params. 

That said, this conversation uncovers a bug. In this case, we'd like GEM2S to run again, but we still want to still use the old GEM2S params. So I thinkn it's better to let `gem2sHash` be null for the case that we want GEM2S to run again, but use the `oldGem2sHash` to save.